.ci/docker/node-playwright/Dockerfile

```diff
@@ -10,7 +10,7 @@ ARG NODEJS_VERSION
 
 # 1. Install node
 RUN apt-get -qq update && apt-get -qq install -y curl && \
-    curl -sL https://deb.nodesource.com/setup_${NODEJS_VERSION}.x | bash - && \
+    curl -sL https://deb.nodesource.com/setup_14.x | bash - && \
```

Comment:
Given that you have set the NODE_OPTIONS on benchmark scripts, can you remove the hardcoded version here?

Reply:
Actually, I will let the hardcoded version here and remove the NODE_OPTIONS from benchmark scripts until we decide to apply this update to the benchmarking, too. My surprise was big when I saw that this dockerfile is also built in a different step (which provides the Node 18 version) of the CI: https://github.com/elastic/apm-agent-rum-js/actions/runs/6284894505/job/17066717885

<img width="996" alt="screenshot" src="https://github.com/elastic/apm-agent-rum-js/assets/15065076/1b03359b-b276-40be-93c1-74632dc05c6a">

And btw, just for more visibility, this is the capture of the Node.js 14 warning I mentioned in the PR's description:

<img width="806" alt="Screenshot 2023-09-26 at 10 40 39" src="https://github.com/elastic/apm-agent-rum-js/assets/15065076/14202fad-fd16-48d7-b30b-4ce1a5d95c50">